### ISSUE TITLE
all plddt value  is zero

### ISSUE THREAD

**Issue by wenyl1919** [Exscientia/abodybuilder3#18]
when i run the abodybuilder3， all plddt values were zero. 

<img width="1483" height="1325" alt="Image" src="https://github.com/user-attachments/assets/94a52def-498d-4719-a860-51828813eea2" />

**Comment by henrykenlay** [comment]
Hi @wenyl1919, are you able to share code for a minimal reproducible example?

**Comment by aloshbau** [comment]
I have the same issue after following 'ABodyBuilder3 Example' in https://github.com/Exscientia/abodybuilder3/blob/main/notebooks/example.ipynb

Here is the code I ran:
```
from abodybuilder3.utils import string_to_input, output_to_pdb, add_atom37_to_output
from abodybuilder3.lightning_module import LitABB3
from abodybuilder3.language.model import ProtT5
import torch

module = LitABB3.load_from_checkpoint("/app/output/plddt-loss/best_second_stage.ckpt")
model = module.model

device = torch.device("cuda" if torch.cuda.is_available() else "cpu")

heavy = "QVQLVQSGAEVKKPGSSVKVSCKASGGTFSSLAISWVRQAPGQGLEWMGGIIPIFGTANYAQKFQGRVTITADESTSTAYMELSSLRSEDTAVYYCARGGSVSGTLVDFDIWGQGTMVTVSS"
light = "DIQMTQSPSTLSASVGDRVTITCRASQSISSWLAWYQQKPGKAPKLLIYKASSLESGVPSRFSGSGSGTEFTLTISSLQPDDFATYYCQQYNIYPITFGGGTKVEIK"

ab_input = string_to_input(heavy=heavy, light=light)
ab_input_batch = {
    key: (value.unsqueeze(0).to(device) if key not in ["single", "pair"] else value.to(device))
    for key, value in ab_input.items()
}
model.to(device)
output = model(ab_input_batch, ab_input_batch["aatype"])
output = add_atom37_to_output(output, ab_input["aatype"].to(device))
pdb_string = output_to_pdb(output, ab_input)
```

and here is a selection from the pdb_string:
```
REMARK   1 CREATED WITH OPENMM 7.7, 2025-12-19
ATOM      1  N   GLN H   0     -13.557  13.693   2.757  1.00  0.00           N  
ATOM      2  H   GLN H   0     -14.744  13.525   2.807  1.00  0.00           H  
ATOM      3  H2  GLN H   0     -13.480  14.881   2.908  1.00  0.00           H  
ATOM      4  H3  GLN H   0     -13.277  13.615   1.592  1.00  0.00           H  
ATOM      5  CA  GLN H   0     -12.758  12.801   3.590  1.00  0.00           C  
ATOM      6  HA  GLN H   0     -11.675  13.048   3.147  1.00  0.00           H  
ATOM      7  C   GLN H   0     -13.307  11.378   3.556  1.00  0.00           C  
ATOM      8  CB  GLN H   0     -12.708  13.311   5.031  1.00  0.00           C  
```

**Comment by aloshbau** [comment]
I have the same issue after following the second example too: 'ABodyBuilder3-LM Example' in https://github.com/Exscientia/abodybuilder3/blob/main/notebooks/example.ipynb

```
from abodybuilder3.utils import string_to_input, output_to_pdb, add_atom37_to_output
from abodybuilder3.lightning_module import LitABB3
from abodybuilder3.language.model import ProtT5
import torch

# Input

heavy = "QVQLVQSGAEVKKPGSSVKVSCKASGGTFSSLAISWVRQAPGQGLEWMGGIIPIFGTANYAQKFQGRVTITADESTSTAYMELSSLRSEDTAVYYCARGGSVSGTLVDFDIWGQGTMVTVSS"
light = "DIQMTQSPSTLSASVGDRVTITCRASQSISSWLAWYQQKPGKAPKLLIYKASSLESGVPSRFSGSGSGTEFTLTISSLQPDDFATYYCQQYNIYPITFGGGTKVEIK"


device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
module = LitABB3.load_from_checkpoint(
    "/app/zenodo/output_loss/language-loss/best_second_stage.ckpt",
    map_location=device      # device
)
model = module.model
model.to(device)              # device

# StructureModule(
#   (linear_in_node): Linear(in_features=1024, out_features=128, bias=True)
#   (linear_in_edge): Linear(in_features=132, out_features=127, bias=True)
#   (ipa_layers): ModuleList(
#     (0-7): 8 x InvariantPointAttention(
#       (linear_q): Linear(in_features=128, out_features=192, bias=True)
#       (linear_kv): Linear(in_features=128, out_features=384, bias=True)
#       (linear_q_points): Linear(in_features=128, out_features=144, bias=True)
#       (linear_kv_points): Linear(in_features=128, out_features=432, bias=True)
#       (linear_b): Linear(in_features=128, out_features=12, bias=True)
#       (linear_out): Linear(in_features=2112, out_features=128, bias=True)
#       (softmax): Softmax(dim=-1)
#       (softplus): Softplus(beta=1, threshold=20)
#     )
#   )
#   (ipa_dropout): Dropout(p=0.1, inplace=False)
#   (layer_norm_ipa_layers): ModuleList(
#     (0-7): 8 x LayerNorm()
#   )
#   (transition_layers): ModuleList(
#     (0-7): 8 x StructureModuleTransition(
#       (layers): ModuleList(
#         (0): StructureModuleTransitionLayer(
#           (linear_1): Linear(in_features=128, out_features=256, bias=True)
#           (linear_2): Linear(in_features=256, out_features=256, bias=True)
#           (linear_3): Linear(in_features=256, out_features=128, bias=True)
#           (relu): ReLU()
#         )
#       )
#       (dropout): Dropout(p=0.1, inplace=False)
#       (layer_norm): LayerNorm()
#     )
#   )
#   (bb_update_layers): ModuleList(
#     (0-7): 8 x BackboneUpdate(
#       (linear): Linear(in_features=128, out_features=6, bias=True)
#     )
#   )
#   (angle_resnet_layers): ModuleList(
#     (0-7): 8 x AngleResnet(
#       (linear_in): Linear(in_features=128, out_features=256, bias=True)
#       (linear_initial): Linear(in_features=128, out_features=256, bias=True)
#       (layers): ModuleList(
#         (0-1): 2 x AngleResnetBlock(
#           (linear_2): Linear(in_features=256, out_features=256, bias=True)
#           (linear_3): Linear(in_features=256, out_features=256, bias=True)
#           (relu): ReLU()
#         )
#       )
#       (linear_out): Linear(in_features=256, out_features=14, bias=True)
#       (relu): ReLU()
#     )
#   )
#   (plddt): PerResidueLDDTCaPredictor(
#     (layer_norm): LayerNorm()
#     (linear_1): Linear(in_features=128, out_features=256, bias=True)
#     (linear_2): Linear(in_features=256, out_features=256, bias=True)
#     (linear_3): Linear(in_features=256, out_features=50, bias=True)
#     (relu): ReLU()
#   )
# )

plm = ProtT5()

embedding = plm.get_embeddings([heavy], [light])[0].to(device)   # device
ab_input = string_to_input(heavy=heavy, light=light)
ab_input["single"] = embedding.unsqueeze(0)                    # device

# 3. Build the batch and double‑check every tensor
ab_input_batch = {}
for key, val in ab_input.items():
    val = val.to(device)                          # move after every key
    if key not in ["single", "pair"]:
        val = val.unsqueeze(0)   # add batch dimension
    ab_input_batch[key] = val

# Inspect -------------------------------------------------------------
print('\nBatch summary (tensor: shape | dtype | device):')
for key, val in ab_input_batch.items():
    print(f'  {key:6s}: {val.shape}, {val.dtype}, {val.device}')

#   is_heavy: torch.Size([1, 229]), torch.int64, cuda:0
#   aatype: torch.Size([1, 229]), torch.int64, cuda:0
#   residue_index: torch.Size([1, 229]), torch.int64, cuda:0
#   single: torch.Size([1, 229, 1024]), torch.float32, cuda:0
#   pair  : torch.Size([1, 229, 229, 132]), torch.float32, cuda:0

# Run the model
output = model(ab_input_batch, ab_input_batch["aatype"])
output = add_atom37_to_output(output, ab_input_batch["aatype"])
pdb_string = output_to_pdb(output, ab_input)

# Warning: importing 'simtk.openmm' is deprecated.  Import 'openmm' instead.
# >>> print(pdb_string[:300])
# REMARK   1 CREATED WITH OPENMM 7.7, 2025-12-19
# ATOM      1  N   GLN H   0      13.910  -6.082  12.284  1.00  0.00           N  
# ATOM      2  H   GLN H   0      12.766  -6.203  12.614  1.00  0.00           H  
# ATOM      3  H2  GLN H   0      14.508  -6.870  12.962  1.00  0.00           H
